src/app/clusters/ota-requestor/OTARequestor.h

```diff
@@ -113,6 +113,8 @@ class OTARequestor : public OTARequestorInterface, public BDXDownloader::StateDe
         app::DataModel::Nullable<uint8_t> percent;
         percent.SetNull();
         OtaRequestorServerSetUpdateStateProgress(percent);
+
+        chip::DeviceLayer::PlatformMgrImpl().AddEventHandler(OnCommissioningCompleteRequestor, reinterpret_cast<intptr_t>(this));
```

Comment:
What do you do if this fails. Init should return a CHIP_ERROR so products can decide what to do if OTA fails to init. `AddEventHandler` is side-effecting and may encounter resource limitations.

Reply:
Done